src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -447,6 +447,8 @@ protected DynamoViewModel(StartConfiguration startConfiguration)
 
             this.BrandingResourceProvider = startConfiguration.BrandingResourceProvider ?? new DefaultBrandingResourceProvider();
 
+            InitializeDelegateCommands();
+
```

Comment:
did ``InitializeDelegateCommands`` need to be moved here, if the order is important please add a comment specifying